src/views/Nft/utils/contracts.ts

```diff
@@ -3,18 +3,18 @@ import { getContract } from 'utils/web3'
 import { ContractOptions } from 'web3-eth-contract'
 import rabbitmintingfarm from 'config/abi/rabbitmintingfarm.json'
 import pancakeRabbits from 'config/abi/pancakeRabbits.json'
-import { RABBIT_MINTING_FARM_ADDRESS, PANCAKE_RABBITS_ADDRESS } from 'config/constants/nfts'
+import { getPancakeRabbitsAddress, getRabbitMintingFarmAddress } from 'utils/addressHelpers'
 
 // TODO: Figure out how to add current account to contracts to write methods can be used
 
 export const getRabbitMintingContract = (contractOptions?: ContractOptions) => {
   const rabbitMintingFarmAbi = (rabbitmintingfarm as unknown) as AbiItem
-  return getContract(rabbitMintingFarmAbi, RABBIT_MINTING_FARM_ADDRESS, contractOptions)
+  return getContract(rabbitMintingFarmAbi, getRabbitMintingFarmAddress(), contractOptions)
 }
 
 export const getPancakeRabbitContract = (contractOptions?: ContractOptions) => {
   const pancakeRabbitsAbi = (pancakeRabbits as unknown) as AbiItem
-  return getContract(pancakeRabbitsAbi, PANCAKE_RABBITS_ADDRESS, contractOptions)
+  return getContract(pancakeRabbitsAbi, getPancakeRabbitsAddress(), contractOptions)
```

Comment:
https://github.com/pancakeswap/pancake-frontend/blob/feature/gamification-profile/src/utils/contractHelpers.ts

Reply:
@RabbitDoge This function has been here for the NFT's. Do you prefer porting it over and use the one you created?